### PR TITLE
🦎 Support metis mainnet

### PR DESCRIPTION
**Pull request by titandac**
add metis mainnet and rename Stardust to MetisStardust to make it more clear it's a Metis testnet.

### CODE REVIEW

File: packages/core/src/constants/chainId.ts [hunk 1]
```diff
@@ -80,7 +82,8 @@ export enum ChainId {
   Avalanche = 43114,
   Songbird = 19,
   MoonbaseAlpha = 1287,
-  Stardust = 588,
+  Metis = 1088,
+  MetisStardust = 588,
```
Comment (tt-marek) [inline packages/core/src/constants/chainId.ts:86]: That is compatibility breaking change, better stay with Stardust...

Comment (titandac) [reply packages/core/src/constants/chainId.ts:86]: that's a good point. new commit incoming.

Comment (titandac) [reply packages/core/src/constants/chainId.ts]: I changed it back to 'Stardust'. Additionally I think it makes more sense to refer to the mainnet chain by it's official name, 'Andromeda'. My understanding is that in the future there may be multiple Metis mainnet chains, therefore calling one of them 'Metis' may become misleading.

**Comment by tt-marek** [comment]
Need lint fix

**Comment by titandac** [comment]
thanks for waiting on this, wasn't able to work on this until today.

fixed the lint issue and squashed into a single commit.

**Review by mj426382 — APPROVED** [review]

**Pull request merged**